security-policy.md

```diff
@@ -1,45 +1,62 @@
-- **Overview:** high-level overview of the issue and self-assessed severity
+This key can also be found at [pgp.mit.edu](https://pgp.surfnet.nl/pks/lookup?op=vindex&fingerprint=on&search=0xE683061E2F85C381).
+
+Also, use the following template when reporting vulnerabilities so that it contains all the required information that helps expedite the analysis and mitigation process.
+
+- **Vulnerable Ballerina artifact(s) and version(s):** list  the vulnerable Ballerina artifact(s) and version(s) 
+- **Overview:** provide a high-level overview of the issue and self-assessed severity
 - **Description:** include the steps to reproduce
-- **Impact:** self-assessed impact
-- **Solution:** any proposed solution
+- **Impact:** state self-assessed impact
+- **Solution:** propose a  solution if you have one
+
+We will keep you informed of the progress towards a fix and disclosure of the vulnerability if the reported issue is identified as a true positive.
+
+## Handling a Vulnerability
+
+Here is an overview of our approach to handling vulnerabilities:
+
+1. The vulnerability will be reported privately to our security team at [security@ballerina.io](mailto:security@ballerina.io).
+
+2. Your email will be acknowledged within 24 hours, and you’ll receive a detailed response to your email indicating the next steps in handling your report. You will be updated on the progress. 
+3. If case of a true-positive finding, the reported vulnerability will be confirmed and fixed by the relevant teams at WSO2.
+4. The fix will be applied to the main branch and a new version of the distribution will be released immediately if required.
+
+
+
+5. The reported user is kept updated on the progress of the process. 
 
-We will keep you informed of the progress towards a fix and disclosure of the vulnerability if the reported issue is identified as a true positive. 
+## Getting Acknowledged and Rewarded
 
-## Handling a vulnerability
+Your efforts in reporting vulnerabilities or any other issues related to the security of Ballerina will be recognized and honored via the [WSO2 Security Reward and Acknowledgement Program](https://docs.wso2.com/display/Security/WSO2+Security+Reward+and+Acknowledgement+Program). 
 
-The below is an overview of the vulnerability handling process.
 
-1. The vulnerability will be reported privately to [security@ballerina.io](mailto:security@ballerina.io). (The initial response time will be less than 24 hours).
-2. The reported vulnerability gets fixed and the solution gets verified by the relevant teams at WSO2.
-3. The fix gets applied to the main branch and a new version of the distribution gets released if required.
-4. The reported user is kept updated on the progress of the process. 
+>**Note:** The reward program is currently applicable to vulnerabilities reported only in the following applications and code repositories:
 
+- [Ballerina website](https://ballerina.io/)
+- [Ballerina platform](https://github.com/ballerina-platform)
```

Comment:
This seemed to be too generic and also feels like everything is covered with this clause. Shall we update as follows (assuming Ballerina central needs to be excluded as per the previous discussion):
```suggestion
>**Note:** The reward program is currently applicable to vulnerabilities reported only in compiler, runtime, CLI tooling, standard libraries, VS Code plugin and [website](https://ballerina.io).
```

Reply:
+1

Reply:
+1

Reply:
Should be `standard library` not libraries.